mypy/test/testpep561.py

```diff
@@ -23,8 +23,8 @@ class PEP561Suite(DataSuite):
     files = ["pep561.test"]
     base_path = "."
 
-    def run_case(self, test_case: DataDrivenTestCase) -> None:
-        test_pep561(test_case)
+    def run_case(self, testcase: DataDrivenTestCase) -> None:
+        test_pep561(testcase)
```

Comment:
#18356 changed the superclass implementation instead. However as this is the only violation, I'm not sure that's necessary here.
https://github.com/python/mypy/blob/d79d89e3ff31ed67a41b0663da9ec8e037d41fa0/mypy/test/data.py#L818-L820